frontend/src/components/layout/ErrorFallback.tsx

```
@@ -0,0 +1,59 @@
+import Lottie from "lottie-react";
+import cableAnimation from '../../animations/cableAnimation.json'
+import React from "react";
+import ButtonLink from "../general/links/ButtonLink";
+import { MoveRight, RefreshCw } from "lucide-react";
+import useStore from "@/store/store";
+
+interface ErrorFallbackProps {
+  error: Error;
+  resetErrorBoundary: () => void;
+}
+
+const ErrorFallback: React.FC<ErrorFallbackProps> = ({ error, resetErrorBoundary }) => {
+  const { isAuthenticated } = useStore((state) => ({
+    isAuthenticated: state.auth.isAuthenticated,
+  }))
+
+  return (
+    <>
+      <figure aria-hidden="true" className="absolute top-0 inset-x-0 z-0">
+        <Lottie className="h-[50vh]" animationData={cableAnimation} />
+      </figure>
+      <div className="mt-[45vh] mx-auto max-w-208 sm:mt-[50vh] xl:max-w-screen-lg">
+        <section className="my-28 px-4 md:px-6 lg:my-36 xl:my-52">
+          <h1 className="font-lato font-bold text-4xl mb-4 lg:mb-6 lg:text-5xl lg:text-center xl:text-6xl">
+            Upps, hier ist etwas schief gegangen!
+          </h1>
+          <p className="lg:text-center mb-10">
+            Folgender Fehler ist passiert: {error.message}.
```

Maybe you can also return the error code? Like 400/500 etc.